lib/glimesh_web/live/user_live/profile.ex

```diff
@@ -20,6 +20,7 @@ defmodule GlimeshWeb.UserLive.Profile do
 
         streamer_share_text = Profile.streamer_share_text(streamer, profile_url)
         viewer_share_text = Profile.viewer_share_text(streamer, profile_url)
+        maybe_channel = if Glimesh.ChannelLookups.get_channel_for_user(streamer), do: Glimesh.ChannelLookups.get_channel_for_user(streamer), else: nil
```

Comment:
```suggestion
        maybe_channel = if channel =  Glimesh.ChannelLookups.get_channel_for_user(streamer), do: channel, else: nil
```

Would be double as fast.

Reply:
Haha I knew there was some fast way of doing it